app/src/main/kotlin/io/github/plastix/kotlinboilerplate/data/remote/ApiModule.kt

```diff
@@ -11,30 +11,22 @@ import javax.inject.Singleton
 class ApiModule {
 
     @Provides @Singleton
-    fun provideApiService(retrofit: Retrofit): GithubApiService {
-        return retrofit.create(GithubApiService::class.java)
-    }
+    fun provideApiService(retrofit: Retrofit) = retrofit.create(GithubApiService::class.java)!!
```

Comment:
To avoid phantom type that is returned by create function, while it is Java function. It can be either solved by not null assertion (!!) or by setting function type. In situations like this, where it is obvious for all experienced Android developers that this will return GithubApiService I prefer to use !! because it is shorter. But I can change to: 
```
fun provideApiService(retrofit: Retrofit): GithubApiService =
        retrofit.create(GithubApiService::class.java)
```
what is also totally reasonable.